packages/test/test-utils/api-report/test-utils.api.md

```diff
@@ -241,11 +241,17 @@ export const mockConfigProvider: (settings?: Record<string, ConfigTypes>) => ICo
 export const retryWithEventualValue: <T>(callback: () => Promise<T>, check: (value: T) => boolean, defaultValue: T, maxTries?: number, backOffMs?: number) => Promise<T>;
 
 // @internal
-export function summarizeNow(summarizer: ISummarizer, inputs?: string | IOnDemandSummarizeOptions): Promise<{
+export function summarizeNow(summarizer: ISummarizer, inputs?: string | IOnDemandSummarizeOptions): Promise<SummaryInfo>;
+
+// @internal
+export interface SummaryInfo {
+    // (undocumented)
```

Comment:
Let's be sure to document the properties here, since this is exported by the package.

Reply:
updated documentation